composer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "unisharp/laravel-filemanager",
+  "name": "uasoft-indonesia/laravel-filemanager",
```

Comment:
This pull request contains namespace change so we cannot merge this. Please undo the namespace change.